tartiflette/sdl/transformers/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .location import Location
```

Comment:
I would rather use the same import mindset on each file.
**Absolute** instead **relative** method.

Reply:
This does not exist anymore.

Reply:
Removed all relative imports everywhere else.

Reply:
Top 👍